test/powershell/Modules/Microsoft.PowerShell.Utility/Format-Custom.Tests.ps1

```diff
@@ -7,20 +7,20 @@ Describe "Format-Custom" -Tags "CI" {
         It "Should have the same output between the alias and the unaliased function" {
             $nonaliased = Get-FormatData | Format-Custom
             $aliased    = Get-FormatData | fc
-            $($nonaliased | Out-String).CompareTo($($aliased | Out-String)) | Should Be 0
+            $($nonaliased | Out-String).CompareTo($($aliased | Out-String)) | Should -Be 0
         }
     }
 
     Context "Check specific flags on Format-Custom" {
 
         It "Should be able to specify the depth in output" {
             $getprocesspester =  Get-FormatData | Format-Custom -depth 1
-            ($getprocesspester).Count                   | Should BeGreaterThan 0
+            ($getprocesspester).Count                   | Should -BeGreaterThan 0
```

Comment:
The extra whitespace between `Count` and the pipe symbol doesn't seem necessary here